src/haddock/libs/libplots.py

```diff
@@ -706,36 +815,125 @@ def clt_table_handler(clt_file, ss_file):
     fig :
         an instance of plotly.graph_objects.Figure
     """
+    # table of statistics
     dfcl = read_capri_table(clt_file)
     statistics_df = clean_capri_table(dfcl)
+
+    # table of structures
     structs_df = find_best_struct(ss_file, number_of_struct=10)
-    structs_df = _add_links(structs_df)
 
     # Order structs by best (lowest score) cluster on top
-    structs_df = structs_df.set_index('Cluster ID')
-    structs_df = structs_df.reindex(index=statistics_df['Cluster ID'])
+    structs_df = structs_df.set_index("Cluster ID")
+    structs_df = structs_df.reindex(index=statistics_df["Cluster ID"])
     structs_df = structs_df.reset_index()
 
-    fig = make_subplots(
-        rows=2,
-        cols=1,
-        specs=[[{"type": "table"}], [{"type": "table"}]],
-        vertical_spacing=0.1,
-        )
-    for i, df in enumerate([statistics_df, structs_df]):
-        table = go.Table(
-            columnwidth=100,
-            header=dict(values=list(df.columns), align="left"),
-            cells=dict(
-                values=df.transpose().values.tolist(), align="left", height=40
-                ),
-            )
-        fig.add_trace(table, row=i + 1, col=1)
-    fig.update_layout(title_text="Summary", height=600)
-    return fig
+    # Add download links and viewer
+    structs_links_df = _add_viewers(structs_df)
+
+    # Merge and convert dataframe to html table
+    df_merged = pd.merge(statistics_df, structs_links_df, on=["Cluster ID"])
+    df_merged = df_merged.set_index("Cluster ID")
+    table = _pandas_df_to_html_table(df_merged.T, table_id="table")
+    return [table]
+
+
+def _css_styles_for_report():
+    custom_css = '''
+    .table {
+        font-family: Arial, sans-serif;
+        border-collapse: collapse;
+        width: 100%;
+        font-size: 16px;
+        height: 500px;
```

Comment:
When more scores are added then you can no longer see the best structures. You will need to scroll down to see them, which a user might not know.

Can the height be made dependent on the number of rows? For example use `12rem` as their are 1 header + 11 score rows + 1 best structure row.

Reply:
Do you mean no vertical scroll in the table? then, I set the `height=auto`.  The table will automatically adjust its height based on the content. Is this what you meant? :thinking: